studia-historica-upsaliensa.csl

```diff
@@ -0,0 +1,588 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" and="symbol" initialize="false" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Studia Historica Upsaliensa </title>
```

Comment:
I changed it. I see the difference between a common university style and a department specific one. So please disregard the question above. Dropped the "department" part of your suggestion. Seems to be inline with the naming of similar department styles in the repos so I hope that is fine with you.